src/gutenberg2zim/export.py

```
@@ -877,6 +887,9 @@ def write_book_presentation_article(
     formats,
 ):
     article_name = article_name_for(book=book, cover=True)
+    if article_name in Global.added_items:
```

Could you please explain why we need this duplicate check ? Maybe an example where this happen is enough. I don't get the rationale behind this, and would like to avoid "mostly silent errors" in the future.

---

You are right lol. This logic was used in my former testing to debug on when Id 5739 generate duplicate copies.